### ISSUE TITLE
YapDatabaseExtensionPrivate.h Redefinition Issues with Xcode 10.2 / Yap 3.1.2

### ISSUE THREAD

**Issue by chrisballinger** [yapstudios/YapDatabase#488]
Rolling back to Yap 3.1.1 resolves the issue.

My guess it has something to do with this change: https://github.com/yapstudios/YapDatabase/commit/2a6474bb58d18de29ac6ad0c4b29944d9996663c#diff-e0f055b4c506985442d1863768c6f92c

**Comment by robbiehanson** [comment]
I think you're running into issue #479 

We tried switching to modulemap. Which solved one problem, but caused another. I think this [commit](https://github.com/yapstudios/YapDatabase/commit/c0c7ec2c4694c4997ab55abc008e73135b32f713) may fix the problem for you, which I discussed in this [comment](https://github.com/yapstudios/YapDatabase/pull/479#issuecomment-455698513).

Let me know if this does the trick, and I'll push a new version to CocoaPods.
